test/ant-test-files/bin/ant

```diff
@@ -0,0 +1,4 @@
+# /bin/bash
+# this file exists to provide an 'executable' to run ant tests
+# it doesn't need to do anything-- it just needs to exist
+echo ':)'
```

Comment:
The test for this fails in the Windows tests because it's actually trying to find `ant.exe` in path. Add a 0 byte `ant.exe` file in this directory too, and I think it should work.

Reply:
Zeroed out the `ant` executable, and added an `ant.exe` file in d1b90eaadc2c15035262d271ecbbe0943498d932